hibernate-core/src/main/java/org/hibernate/sql/exec/internal/JdbcParameterImpl.java

```diff
@@ -13,6 +13,7 @@
  * @author Steve Ebersole
  */
 public class JdbcParameterImpl extends AbstractJdbcParameter {
+
 	public JdbcParameterImpl(JdbcMapping jdbcMapping) {
```

Comment:
I think it is one of @sebersole 's personal styles to leave out the blank line before the first element statement in a class. I prefer the above change but it is not a big deal.

Another style of Steve is to leave out the space delimiter between generic types as following:
```
Map<K,V> map; // I would prefer Map<K, V> map
```
But I don't think it is a big deal and they have become my style components as well, :).